sdks/python/container/Dockerfile

```diff
@@ -82,6 +78,32 @@ RUN  \
     # Remove pip cache.
     rm -rf /root/.cache/pip
 
+FROM gcr.io/distroless/cc-debian12 as beam
+LABEL Author "Apache Beam <dev@beam.apache.org>"
+ARG TARGETOS
+ARG TARGETARCH
+
+# copy commands & libs to distroless
+COPY --from=python-base /bin /bin
+COPY --from=python-base /lib lib
+COPY --from=python-base /usr/bin/which /usr/bin/which
+COPY --from=python-base /etc/ld.so.cache /etc/ld.so.cache
+COPY --from=python-base /usr/bin/which /usr/bin/which
+
+# copy packages to distroless
+COPY --from=python-base /usr/local/lib /usr/local/lib
+COPY --from=python-base /usr/local/gcloud /usr/local/gcloud
+COPY --from=python-base /usr/local/bin /usr/local/bin
+
+ENV PATH="/usr/local/bin:/usr/local/gcloud/google-cloud-sdk/bin:$PATH"
+
+COPY target/base_image_requirements.txt /tmp/base_image_requirements.txt
```

Comment:
Line 100-101 shouldn't be necessary, this was necessary around line 72, in a different image.